fitbenchmarking/fitbenchmarking
# Change name of `sasview` controller

#### Description of Work

Fixes #536 

Changes name of `sasview` controller to a `bumps` controller


#### Testing Instructions

1. Check that `sasview` is only mentioned in the parsers and not the controllers

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
